src/main/java/banana/Duke.java

```diff
@@ -0,0 +1,352 @@
+package banana;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDate;
+
+/**
+ * The Duke class runs the duke/chatbot
+ * program.
+ *
+ * @author: Ravi Ananya
+ **/
+
+public class Duke {
+
+    private static final String WELCOME_LABEL = "Hello! I'm Banana \n" + "     What can I do for you?";
+    private static final String BYE_LABEL = "Bye. Hope to see you again soon!";
+
+    private Storage store;
```

Comment:
Perhaps it will be better if a noun was used for this variable name, instead of using a verb (which is better suited for methods/functions) ? 
```suggestion
    private Storage storage;
```